javaclient/src/main/java/confluo/streaming/ConfluoProducer.java

```diff
@@ -0,0 +1,157 @@
+package confluo.streaming;
+
+import confluo.rpc.*;
+import confluo.streaming.config.ProducerConfig;
+import java.io.Closeable;
+import org.apache.thrift.TApplicationException;
+import org.apache.thrift.TException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.Properties;
+
+/**
+ * ConfluoProducer will create the topic if it isn't existed,or reuse the old topic. batch produce
+ * message can be enable if batch size was set to bigger than 1,not thread safe
+ */
+public class ConfluoProducer implements Closeable {
+  Logger logger = LoggerFactory.getLogger(ConfluoProducer.class);
+  private String topic;
+  private String host;
+  private int port;
+  private long totalProducedNum;
+  private int produceBatchSize;
+  private boolean batchEnable;
+  private int messageSize;
+  private rpc_storage_mode storageMode;
+  private String produceSchema;
+  private RpcClient client;
+  private MessageBatchBuilder batchBuilder;
+  private Schema curSchema;
+  private int batched;
+
+  public ConfluoProducer(Properties properties) throws TException {
+    host =
+        properties.getProperty(
+            ProducerConfig.BOOTSTRAP_ADDRESS, ProducerConfig.BOOTSTRAP_ADDRESS_DEFAULT);
+    port =
+        Integer.valueOf(
+            properties.getProperty(
+                ProducerConfig.BOOTSTRAP_PORT, ProducerConfig.BOOTSTRAP_PORT_DEFAULT));
+    topic = properties.getProperty(ProducerConfig.TOPIC, ProducerConfig.TOPIC_DEFAULT);
+    storageMode =
+        rpc_storage_mode.valueOf(
+            properties.getProperty(
+                ProducerConfig.ARCHIVE_MODE, ProducerConfig.ARCHIVE_MODE_DEFAULT));
+    messageSize =
+        Integer.valueOf(
+            properties.getProperty(
+                ProducerConfig.MESSAGE_SIZE, ProducerConfig.MESSAGE_SIZE_DEFAULT));
+    produceSchema = createSchema(messageSize - 8);
+    produceBatchSize =
+        Integer.valueOf(
+            properties.getProperty(ProducerConfig.BATCH_SIZE, ProducerConfig.BATCH_SIZE_DEFAULT));
+    batchEnable = produceBatchSize > 1 ? true : false;
+    start();
+    logger.info(
+        String.format(
+            "\n ------------------- "
```

Comment:
It would be good if the string is better formatted (consistent capitalization, whitespaces, etc.); this comment extends to all log messages in general..

Reply:
i really don't know whtat kind of format it's should be ,to solve the formatting issue, i try to reformat it by google-java-format plugin, you may help me fix formatting issue